RecoTauTag/Configuration/python/RecoPFTauTag_cff.py

```diff
@@ -115,3 +115,7 @@
 PFTau = cms.Sequence(
     PFTauTask
 )
+
+from Configuration.Eras.Modifier_pp_on_XeXe_2017_cff import pp_on_XeXe_2017
+PFTauDummy=cms.Sequence()
+pp_on_XeXe_2017.toReplaceWith(PFTau,PFTauDummy)
```

Comment:
was this sequence problematic to run?

Reply:
It takes a long time and we won't use taus

Reply:
do you care about miniAOD overall for this processing?

Reply:
I would not completely rule out miniAOD being useful, in case it's written.  It's in the relval wf and appears to be produced without any issue.

Reply:
steps in the 148.0 wf "RAW2DIGI,L1Reco,RECO,EI,VALIDATION,DQM" 
do not include miniAOD
.. also, EI fails without tau inputs [or at least I think it does on more events running]

Reply:
I'm not sure what you mean by "steps in the wf including mini-AOD".  
Which step is missing?

Reply:
miniAOD step is called "PAT". I don't see it in the new pp_on_XeXe workflows